common/src/main/java/io/pravega/common/concurrent/MultiKeySequentialProcessor.java

```diff
@@ -85,7 +85,7 @@ public int getCurrentTaskCount() {
      * In addition, no subsequent task for any of the given dependency Keys will begin executing until this task has
      * finished executing.
      *
-     * @param keys         A Collection of {@link KeyType} objects representing the Keys that this task is dependent on.
+     * @param keys         A Collection of {@literal {@link KeyType}} objects representing the Keys that this task is dependent on.
```

Comment:
Won't this print "{@link KeyType}" ?

Reply:
um.... yes. probably. javadoc under 8 won't recognize the KeyType type even though the code does.  Instead I could remove the @link since there isn't really any reference around for it to link to. You caught me on that one.  I'll try it with the link in place under a java 10 compile to see if that helps.

Reply:
Removed the literal and left just the reference to KeyType without the link since there isn't a valid link for it.